src/main/java/org/opentripplanner/raptor/rangeraptor/standard/ArrivalTimeRoutingStrategy.java

```diff
@@ -71,7 +71,7 @@ public void prepareForTransitWith(RaptorTimeTable<T> timeTable) {
   }
 
   @Override
-  public void alight(final int stopIndex, final int stopPos, final int alightSlack) {
```

Comment:
Did you remove them for readability or what was the reason?

Reply:
Using `final` for method arguments is not necessary. It has no purpose, at least not any more in Java 17. The final can be inferred by the compiler - but it does not have an effect on performance or any thing else, so I believe it is not. This is just left over from the time where we used final in every place possible, but it only have an effect on fields, not method arguments.